packages/taco/src/taco.ts

```diff
@@ -202,6 +185,7 @@ export const isAuthorized = async (
     messageKit,
   );
 
+// TODO is this still valid and actually needed? should we remove this?
```

Comment:
I think it's worthwhile to revisit it post release of `@nucypher/contracts` - IIRC, there were some changes to `EncryptorAllowlist` contract, etc.

So we may want to revisit how (if) to expose this logic in `taco-web`.